src/navigation/appNavigation.js

```diff
@@ -283,7 +290,6 @@ const assetsFlow = createStackNavigator(
     [ASSET]: AssetScreen,
     [COLLECTIBLE]: CollectibleScreen,
     [CONTACT]: ContactScreen,
-    [SETTINGS]: SettingsScreen,
```

Comment:
`[RECOVERY_SETTINGS]: RecoverySettingsScreen` and  `[SECURITY_SETTINGS]: SecuritySettingsScreen`, is needed instead. So user could navigate back to Assets screen after pressing on "Backup wallet" or "Enable biometrics login" option in Insight.